src/index.js

```diff
@@ -97,6 +102,13 @@ async function run() {
 
       if (!skipCommit) {
         // Add changed files to git
+        if (preCommit) {
+          await require(preCommit).preCommit({
+            workspace: process.env.GITHUB_WORKSPACE,
```

Comment:
Would this be needed? Since the test file is required in the same env the script could if it needed get the variable itself.

Reply:
Yep, done